SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,63 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Competition Details
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
+
+The event is open to the public, so you're welcome to bring family and friends as guests, however please note that space in the pits is limited.
+
+We'll be providing rules for behaviour in/around the competition venue when you arrive. If you have any questions or concerns beforehand, please let us know.
+
+If you have any issues on the day, we'll be monitoring the `teams@` email address and our Discord server.
```

Comment:
Question: Will we? Most of us will have a lot going on. In the past, we've strongly recommended (and preferred) people use helpdesk over email / Discord.

```suggestion
```

Reply:
I think this is more for people who are arriving late/before they get there rather than within the venue. Agree that Helpdesk is better suited once they're in the venue.

Reply:
8e5c023